src/sections/Learn-Layer5/Chapters/index.js

```diff
@@ -3,17 +3,72 @@ import { MDXRenderer } from "gatsby-plugin-mdx";
 import { SRLWrapper } from "simple-react-lightbox";
 import { Container, Row, Col } from "../../../reusecore/Layout";
 import TOC from "../../../components/Learn-Components/TOC-Chapters";
+import Image from "../../../components/image";
 import { ChapterWrapper } from "./chapters.style";
+import ReactTooltip from "react-tooltip";
 
-const Chapters = ({chapterData, courseData, location}) => {
+const Chapters = ({chapterData, courseData, location, serviceMeshesList}) => {
   const { frontmatter, body } = chapterData;
+  const serviceMeshImages = courseData.frontmatter.meshesYouLearn;
+  
+  const replaceSlugPart = (index) => (oldSlug) => (replacement) => {
+    let parts = oldSlug.split("/");
+    parts[index] = replacement;
+    return parts.join("/");
+  };
+    
+  const replaceServiceMeshInSlug = replaceSlugPart(3)(chapterData.fields.slug);
+
+
+  const isMeshActive = (sm) => chapterData.fields.slug.split("/")[3] === sm;
+
+  const getAvailableServiceMeshes = () => {
+    let serviceMeshes = [];
+    serviceMeshesList.forEach(sm => {
+      if(!serviceMeshes.map(sm => sm.section).includes(sm.fields.section)) 
+        serviceMeshes.push({section: sm.fields.section, slug: replaceServiceMeshInSlug( sm.fields.section)}); 
+    });
+    return serviceMeshes;
+  };
+
+  const findServiceMeshImage = (images, serviceMesh) => images.find(image => image.name.toLowerCase() == serviceMesh);
+
+  const ServiceMeshesAvailable = ({serviceMeshes}) => serviceMeshes.map((sm, index) => 
+    <>
+      <a  href={`/${sm.slug}`} data-for="mesh-name" data-tip={sm.section} className="course" key={sm+index}>
+        <div className={`service-mesh-image ${isMeshActive(sm.section) ? "service-mesh-image-active" : ""}`}>
+          <Image
+            {...findServiceMeshImage(serviceMeshImages, sm.section).imagepath}
+            className="docker"
+            alt={sm.section}
+          />
+        </div>
+      </a>
+      <ReactTooltip 
+        id="mesh-name"
+        place="bottom"
+        effect="solid"
+        backgroundColor="rgb(60,73,79)"
+        className="mesh-tooltip"
+      />
+    </>
+  ); 
+
+
+  
+
   return (
     <ChapterWrapper>
       <Container className="chapter-container">
         <Row>
           <Col sm={12} md={3}>
             <TOC courseData={courseData} chapterData={chapterData} location={location} />
```

Comment:
We need to update this component as well, to update the paths in the TOC, for the selected service mesh. An `useState` defined in this component can help us achieve that.

Reply:
I see, i missed it, will do